Rubberduck.Core/AutoComplete/Service/SelfClosingPairCompletionService.cs

```diff
@@ -168,6 +227,34 @@ public MatchingTokenVisitor(SelfClosingPair pair, CodeString code)
                 _code = code;
             }
 
+
+            public override Selection VisitArgumentList(VBAParser.ArgumentListContext context)
+            {
+                if (context.Start.Text.StartsWith(_pair.OpeningChar.ToString())
+                    && context.Start.Text.EndsWith(_pair.ClosingChar.ToString()))
+                {
```

Comment:
I don't think this condition is appropriate. Is this a copy paste error from the literal expression?

Why should the first and last character of the first token be a closing pair, unless that token is a literal anyway? Why do other tokens not matter?

Reply:
@MDoerner yeah this whole override shouldn't even be there, since `ArgumentList` won't have the parens anyway. I'll remove it, but then short of having `LPAREN`/`RPAREN` tokens on the `CallStmtContext` I'm not sure what's missing; the input string isn't parsing into a well-formed parse tree, so it's possible _nothing_ is missing from this visitor and `VisitArgumentList` should just be removed.